Ch02/Ch02.md

```diff
@@ -274,8 +273,7 @@ a simple console application to explore the RavenDB API. Typically, RavenDB is u
 so we'll also explore some of the common patterns of organizing your RavenDB usage in your application later in 
 this chapter.
 
-The code samples in this book use C#, but the documentation can guide you on how to achieve the same results in any
-supported clients. 
+The code samples in this book use C#, but the documentation can guide you on how to achieve the same results with any supported client.  
```

Comment:
Should probably change it to "most of the cost samples"

Reply:
done